src/PyCall.jl

```diff
@@ -235,9 +240,9 @@ function show(io::IO, o::PyObject)
 end
 
 function Base.Docs.doc(o::PyObject)
-    if haskey(o, "__doc__")
+    if hasproperty(o, "__doc__")
         d = o["__doc__"]
-        if d.o != pynothing[]
+        if PyPtr(d) != pynothing[]
```

Comment:
Do we want `≛̸` (`\starequal\not`) to handle this?

(I wish `a !≛ b` automatically lowered to `!(≛)(a, b)`...)

Reply:
Probably clearer to just use `!(a ≛ b)`